apps/src/templates/sectionProgress/progressTables/ProgressTableContentView.jsx

```diff
@@ -166,7 +168,8 @@ export default class ProgressTableContentView extends React.Component {
           rowKey={'id'}
           onScroll={this.props.onScroll}
           style={{
-            overflow: 'auto',
+            overflowX: 'scroll',
```

Comment:
i tried just setting this to `scroll` for both to avoid the need for any gutter calculations, but deemed the visual benefit of only showing it conditionally to be well worth the cost.

Reply:
In case this was confusing to anyone else, this comment applies to L171-172 and explains why `overflowY` is not set to `'scroll'`